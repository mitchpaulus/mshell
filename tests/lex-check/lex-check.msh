# A multi line comment
'asdfa

asdfasdf

asdfas'

"another
multi line"

`multi
line path`

'asdfasdf' toInt 9 +
