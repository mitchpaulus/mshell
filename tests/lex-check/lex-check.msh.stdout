Tokens:
2:1:SINGLEQUOTESTRING 'asdfa

asdfasdf

asdfas'
8:1:STRING "another
multi line"
11:1:PATH `multi
line path`
14:1:SINGLEQUOTESTRING 'asdfasdf'
14:12:LITERAL toInt
14:18:INTEGER 9
14:20:PLUS +
15:1:EOF 
